src/shared/vendor/third_party/webcomponents.js

```diff
@@ -6374,4 +6374,6 @@ CustomElements.addModule(function(scope) {
   window.Platform = scope;
 })(window.WebComponents);
 
-module.exports = window.WebComponents;
+if (typeof module !== 'undefined') {
+  module.exports = window.WebComponents;
+}
```

Comment:
For the unit tests run in debug mode, we load webcomponents in the wrapper html and module isn't defined.